### DISCUSSION TITLE
Docker version

### DISCUSSION THREAD

**Discussion by NeoPlayer13** [fifthsegment/Gatesentry#78] [Ideas]
Can you please add a docker version of Gatesentry?

**Comment by fifthsegment** [comment]
Sure, yes! I can dockerize it :)

**Reply by fifthsegment** [reply]
Gatesentry is now dockerized @NeoPlayer13 : https://github.com/fifthsegment/Gatesentry#method-1-using-docker

**Reply by NeoPlayer13** [reply]
Thanks you very much, running perfectly. I'm going to configure and test it.
